### ISSUE TITLE
False-positives for tailwind utilities (only visible in Jetbrains IDE)

### ISSUE THREAD

**Issue by thgla** [oxc-project/oxc-intellij-plugin#576]
I'm getting some false-positive warnings, but only with the IntelliJ plugin - everything looks fine in VS Code or when I run the linter through command line.

Here is an example:

```html
<input
      className="w-80 rounded border border-gray-300 bg-white p-2 dark:border-gray-500 dark:bg-gray-700"
      type="text"
      placeholder="Username"
      value={userName}
      onChange={(e) => setUserName(e.target.value)}
    />
```

And it complains with warnings like these in the IDE:
```
'dark:border-gray-500' applies the same CSS properties as 'border-gray-300'.
'dark:bg-gray-700' applies the same CSS properties as 'bg-white'.
'border-gray-300' applies the same CSS properties as 'dark:border-gray-500'.
```

**Configuration**

`.oxfmtrc`

```json
{
  "$schema": "./node_modules/oxfmt/configuration_schema.json",
  "ignorePatterns": ["src/api/LeafWS/**", "src/api/UstService/**", "dist/**"],
  "singleQuote": true,
  "printWidth": 120,
  "sortTailwindcss": {
    "stylesheet": "./src/index.css",
    "functions": ["clsx", "cn"],
    "preserveWhitespace": true
  }
}
```

`.oxlintrc`

```json
{
  "$schema": "./node_modules/oxlint/configuration_schema.json",
  "plugins": ["react", "typescript", "oxc"],
  "rules": {
    "react/rules-of-hooks": "error",
    "react/only-export-components": ["warn", { "allowConstantExport": true }]
  }
}
```

I would like to share a sample project, but corporate firewall blocks it again. Let me know if it helps, I can create one from home.
Everything is up-to-date: oxfmt@0.65.0, oxlint@1.80.0, vite@8.2.2

Thanks
Thomas

**Comment by thgla** [comment]
This issue is in the Tailwind language server or how IntelliJ integrates it, so not related to this plugin.
